<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradesButtons</class>
 <widget class="QDialog" name="GradesButtons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brainstorm | Grade Action</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageAddGrade">
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>160</y>
         <width>361</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutUserID">
            <item>
             <widget class="QLabel" name="labelUserID">
              <property name="text">
               <string>User ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditUserID"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutSubject">
            <item>
             <widget class="QLabel" name="labelSubject">
              <property name="text">
               <string>Subject</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditSubject"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutGrade">
            <item>
             <widget class="QLabel" name="labelGrade">
              <property name="text">
               <string>Grade</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditGrade"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddGrade">
          <property name="text">
           <string>Add Grade</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="pageUpdateGrade">
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>190</y>
         <width>361</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutGradeID">
            <item>
             <widget class="QLabel" name="labelGradeID">
              <property name="text">
               <string>GradeID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditGradeID"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutNewGrade">
            <item>
             <widget class="QLabel" name="labelNewGrade">
              <property name="text">
               <string>New Grade</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditnewGrade"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonUpdateGrade">
          <property name="text">
           <string>Update Grade</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="pageDeleteGrade">
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>210</y>
         <width>361</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutGradeIDdel">
            <item>
             <widget class="QLabel" name="labelGradeIDdel">
              <property name="text">
               <string>GradeID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditGradeIDdel"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonDeleteGrade">
          <property name="text">
           <string>Delete Grade</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
